proto/stride/stakeibc/params.proto

```diff
@@ -27,6 +27,7 @@ message Params {
   uint64 ibc_transfer_timeout_nanos = 16;
   uint64 validator_slash_query_threshold = 19;
   uint64 validator_weight_cap = 20;
+  uint64 max_ica_messages_per_tx = 21;
```

Comment:
Is it per tx or per block?  Before I thought that it was about the block size how many would fit simultaneously.  Can't more than one tx happen in a block but from the chains point of view these would still be essentially happening at the same time?

Reply:
the limit (we believe) is a gas limit for a single tx. Each ICA is one tx that's built up of many messages - the more messages, the more gas it uses